flipt-client-kotlin-android/README.md

```diff
@@ -0,0 +1,132 @@
+# Flipt Client Kotlin Android
+
+[![Maven Central](https://img.shields.io/maven-central/v/io.flipt/flipt-client-java?label=flipt-client-java)](https://central.sonatype.com/artifact/io.flipt/flipt-client-java)
+
+The `flipt-client-kotlin-android` library contains the Kotlin for Android source code for the Flipt [client-side evaluation](https://www.flipt.io/docs/integration/client) client.
+
+## Installation
+
+### Gradle
+
+Add the dependency in your `build.gradle`:
+
+```groovy
+dependencies {
+    implementation 'io.flipt:flipt-client-kotlin-android:0.x.x'
+}
+```
+
+### Maven
+
+Add the dependency in your `pom.xml`:
+
+```xml
+<dependency>
+    <groupId>io.flipt</groupId>
+    <artifactId>flipt-client-kotlin-android</artifactId>
+    <version>0.x.x</version>
+</dependency>
+```
+
+## How Does It Work?
+
+The `flipt-client-kotlin-android` library is a wrapper around the [flipt-engine-ffi](https://github.com/flipt-io/flipt-client-sdks/tree/main/flipt-engine-ffi) library.
+
+All evaluation happens within the SDK, using the shared library built from the [flipt-engine-ffi](https://github.com/flipt-io/flipt-client-sdks/tree/main/flipt-engine-ffi) library.
+
+Because the evaluation happens within the SDK, the SDKs can be used in environments where the Flipt server is not available or reachable after the initial data is fetched.
+
+## Data Fetching
+
+Upon instantiation, the `flipt-client-kotlin-android` library will fetch the flag state from the Flipt server and store it in memory. This means that the first time you use the SDK, it will make a request to the Flipt server.
+
+### Polling (Default)
+
+By default, the SDK will poll the Flipt server for new flag state at a regular interval. This interval can be configured using the `fetchMode` option when constructing a client. The default interval is 120 seconds.
+
+### Streaming (Flipt Cloud Only)
+
+[Flipt Cloud](https://flipt.io/cloud) users can use the `streaming` fetch method to stream flag state changes from the Flipt server to the SDK.
+
+When in streaming mode, the SDK will connect to the Flipt server and open a persistent connection that will remain open until the client is closed. The SDK will then receive flag state changes in real-time.
+
+## Supported Architectures
+
+This SDK currently supports the following OSes/architectures:
+
+- Linux x86_64
+- Linux x86_64 (musl)
+- Linux arm64
+- Linux arm64 (musl)
+- MacOS x86_64
+- MacOS arm64
+- Windows x86_64
+
+### Glibc vs Musl
+
+Most Linux distributions use [Glibc](https://en.wikipedia.org/wiki/Glibc), but some distributions like Alpine Linux use [Musl](https://en.wikipedia.org/wiki/Musl). If you are using Alpine Linux, you will need to install the `musl` version of the client.
+
```

Comment:
```suggestion
```

I dont think we are doing a MUSL version

Reply:
agree on this.